src/sagas/wallet.js

```diff
@@ -430,7 +430,15 @@ export function* handleTx(action) {
   }
 
   // find tokens affected by the transaction
-  const balances = yield call(wallet.getTxBalance.bind(wallet), tx);
+  const affectedTokens = new Set();
+
+  for (const output of tx.outputs) {
```

Comment:
I explored a little how we could do this and the problem right now is that we don't have a `isAddressMine` method working with the wallet-service facade, and even if we did, it would involve a request to a lambda to check

So I suggest we accept that we could be downloading more data for now and open a KTLO to handle this better later.

What do you think?

Reply:
https://github.com/HathorNetwork/hathor-wallet/issues/355

Reply:
Yeah, the wallet service implementation won't work 100% without this. We can do this in another PR but we should do it before rolling out